css.css

```diff
@@ -15,8 +15,10 @@ body {
 
 .page .header {
```

Comment:
Dear Mr Tu!
I don't understand in this case. Can you help me for make it detail?